src/odemis/driver/hamamatsurx_simcam.py

```diff
@@ -0,0 +1,552 @@
+# -*- coding: utf-8 -*-
+'''
+Created on 29 Oct 2018
+
+@author: Sabrina Rossberger
+
+Copyright © 2018 Sabrina Rossberger, Delmic
+
+This file is part of Odemis.
+
+Odemis is free software: you can redistribute it and/or modify it under the terms of the GNU General Public License version 2 as published by the Free Software Foundation.
+
+Odemis is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License along with Odemis. If not, see http://www.gnu.org/licenses/.
+'''
+
+from __future__ import division
+
+import Queue
+import logging
+from odemis import model, util, dataio
+from odemis.model import oneway
+import time
+import numpy
+import os
+
+
+class ReadoutCamera(model.DigitalCamera):
+    """
+    Represents Hamamatsu readout camera.
+    """
+
+    def __init__(self, name, role, parent, spectrograph=None, daemon=None, **kwargs):
+        """ Initializes a fake Hamamatsu OrcaFlash readout camera.
+        :parameter name: (str) as in Odemis
+        :parameter role: (str) as in Odemis
+        :parameter parent: class streakcamera
+        """
+        # TODO image focus and operate mode
+        # get the fake images
+        try:
+            image = kwargs.pop("image")
+            image = unicode(image)
+            # ensure relative path is from this file
+            if not os.path.isabs(image):
+                image = os.path.join(os.path.dirname(__file__), image)
+            converter = dataio.find_fittest_converter(image, mode=os.O_RDONLY)
+            self._img = []
+            for i in range(10):
+                img = converter.read_data(image)[i+2]
```

Comment:
The file requires 12 images at least, right? I'd suggest to just accept any file, and read all the images in the file. If there is only 1 image, the camera will simulate just a single image. If there are more images, it will display them one at a time.
Also, don't skip the first 2 images. If you don't want the first 2 images, don't put them in your file.
Also, only call `converter.read_data()` once, before the loop, and then iterate over the data.

Reply:
The advantage of dropping the first 2 images is, that we can use any acquisition from the streak camera as input image for the simulator without explicitly modifying the content of the image. Would that not be handy?

Reply:
Yes, that's what I had understood. However, it's making both the code and the behaviour more complicated. In particular, the current code doesn't accept simple files with just one image, which is confusing. In general, you want to keep everything simple.

If one day you need to easily start a simulator with any acquisition, you could write a script that automatically drops the non-streakcam images and write another acquisition file. So, I'd rather choose simplicity for now.

Reply:
Okay dropped the first two images in the example data. Should I attach the example image to the pr?